scripts/ci-check.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+yarn prettier:diff
+yarn lint
+yarn test --runInBand
+cp src/internal/FeatureFlags.js src/internal/FeatureFlags.js.orig
+sed -i -r 's/export *const *(components|breakingChanges)X *= *false/export const \1X = true/g' src/internal/FeatureFlags.js
```

Comment:
```suggestion
sed -i -r 's/export *const *\(components|breakingChanges\)X *= *false/export const \1X = true/g' src/internal/FeatureFlags.js
```

I know the `-r` flag enables extended regex but no capture groups were being returned unless I escaped the parentheses when I tried running this. Not sure if I'm going down the right path with this correction but wanted to see if you also got errors when running the CI shell script locally

Reply:
should we also ignore the output `.js-r` file?

Reply:
@emyarod Have you tried `-E` instead of `-r`? From my research Mac and Linux seems to have different argument for regular expression. BTW the interim backup file will be erased before CI finishes.

Reply:
@asudoh I first tried with the `-E` flag before escaping the parentheses, but I received the same error: `sed: 1: "s/export *const *(comp ...": \1 not defined in the RE`

also thanks for clarifying on the backup file

Reply:
@emyarod I see Mac won't let me use `-i` without back-up file extension. Given it seems that you are reporting issues with Mac and I'm testing with CI environment (Linux), I changed `ci-check.sh` to support both.